### PR TITLE
Eliminate 2 gpu ops during sampling when logit_bias is zero

### PR DESCRIPTION
**Pull request by Qubitium**
Reason for PR: Eliminate 2 gpu ops during sampling when `logit_bias` is not used (all zeros)

1. skip allocation of `logit_bias`
2. skip applying `logits.add_(self.logit_bias)`

On our internal test of Yi-6B quantized using `marlin` on 4090 we see tangible improvement: up to 63% throughput improvement for small sized tokens output work loads when batch > 1.

Concurrency is 10 requests so it will cause sglang to use batch size between 1-10:

### CODE REVIEW

File: python/sglang/srt/managers/router/infer_batch.py [hunk 1]
```diff
@@ -458,15 +463,20 @@ def merge(self, other):
...
-            setattr(
-                self, item, torch.concat([getattr(self, item), getattr(other, item)])
-            )
+            self_val = getattr(self, item, None)
+            other_val = getattr(other, item, None)
+            # logit_bias can be None
+            if self_val is not None and other_val is not None:
+                setattr(
+                    self, item, torch.concat([self_val, other_val])
+                )
```
Comment (hnyls2002) [inline python/sglang/srt/managers/router/infer_batch.py]: How about only one logit bias is None, then we will lose one of them?

Comment (Qubitium) [reply python/sglang/srt/managers/router/infer_batch.py]: @hnyls2002  fixed

**Comment by hnyls2002** [comment]
@Qubitium Sorry to cause trouble. I have directly pushed the commit on your branch.
closes #344
